Automattic/wp-calypso
# Add new comment highlight

**Built on top of https://github.com/Automattic/wp-calypso/pull/40854**

#### Changes proposed in this Pull Request
Add highlight effect to new realtime comments https://d.pr/v/i6dnYz, most likely we will have to update this when updated/deleted comments will have realtime support.

#### Testing instructions
- Apply patch D41517-code
- Sandbox one of your blogs, follow it and start a conversation (multiple comments and replies)
- Add a comment to one of your posts on the frontend, it should be highlighted in calypso reader
